microsoft-edge/devtools-guide-chromium/sample-code/sample-code.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
@captainbrosset - Please review this new page.  
https://review.docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/sample-code/sample-code?branch=pr-en-us-1683

Reply:
Page looks good, and I agree that column 2 isn't really necessary here.